docker-compose.yml

```diff
@@ -11,7 +11,11 @@ services:
     build: elasticsearch
     env_file: .env
     volumes:
-      - elasticsearch:/usr/share/elasticsearch/data
+      - elasticsearch:/usr/share/elasticsearch/data:rw
+    environment:
```

Comment:
Why not use the environment variable file (`.env`)? It's already defined at the line 12.